configuration/go/http/order-processor/go.mod

```diff
@@ -0,0 +1,5 @@
+module dapr_example
+
+go 1.18
```

Comment:
1.19?

Reply:
all other quickstarts are 1.18.  Is there a need or benefit to forcing version higher?  @mukundansundar

Reply:
@paulyuk This is mainly to remain in step with all other Go repos in the Dapr project. 
Additionally we can make use of latest fixes and features in golang if any.